gcovr/__main__.py

```diff
@@ -2309,7 +2308,9 @@ def build_filter(regex):
         return re.compile(os.path.realpath(regex))
 
 
-def main(options, args):
+def main():
+    global options, args
```

Comment:
Is there a specific reason why options needs to be global? While that's technically a precise translation of the status quo, that isn't the semantics I intended.

Reply:
There are still some functions that are expecting those variables to be defined at the global scope. Without this, `gcovr` fails.

Reply:
OK, we will fix that in a future commit when different modules are extracted.